interface-definitions/https.xml.in

```diff
@@ -60,6 +60,39 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <node name="allow-client">
```

Comment:
Can we use it as a separate XML file and include it. So we can re-use this for other services
```
#include <include/allow-client.xml.i>
```

Reply:
But I'm happy to make it an include file, if you prefer.

Reply:
Sure, I'll move it to include.

Reply:
Done; rebased over current.

Reply:
Thanks!